CodeFormatter/phpoptions.cpp

```diff
@@ -28,6 +34,5 @@ PhpOptions& PhpOptions::Load()
 
 PhpOptions& PhpOptions::Save()
 {
-    clConfig config("php.conf");
```

Comment:
The phpoptions.cpp is there to allow the plugins to use chare the phpexec configuration so that the user only has to set it up once instead of in 3 different places to make each plugin work.

The plugins have there own config file that handels there specific settings.